.gitignore

```diff
@@ -3,20 +3,14 @@ node_modules
 tmp
 
 # ignore js, map, and css in element dirs
+elements/*/_temp
```

Comment:
May I ask why `_temp` instead of `.temp`?  It seems to me that making the dir hidden would be cleaner.

Reply:
I think there's value in seeing those folders and the dot naming structure is harder to engage with in a GUI because they're hidden from the user's view.

Reply:
Finder for example, a user won't even see that the folder exists on a Mac; what if I'm debugging and want to look at the compiled asset.  The only way to do that then becomes the command line and I don't want to lock out users.